ci/ray_ci/tester.py

```diff
@@ -157,6 +155,7 @@ def _get_test_targets(
     except_tags: Optional[str] = "",
     only_tags: Optional[str] = "",
     yaml_dir: Optional[str] = None,
+    get_flaky_tests: bool = False,
```

Comment:
update the docs?

maybe use a enum? like `NORMAL_TEST`, `FLAKY_TESTS` (so that you can add an `ALL_TESTS` also in the future)?